trigger/task.go

```diff
@@ -558,6 +559,22 @@ func isTaskStatusRegression(oldStatus, newStatus string) bool {
 	return false
 }
 
+func (t *taskTriggers) testMatchesRegex(testName string, sub event.Subscription) bool {
```

Comment:
Nit, this doesn't need to be a method on taskTriggers, and you shouldn't be copying the whole subscription structure.

Reply:
fixed